packages/react-native/package.json

```diff
@@ -85,7 +86,7 @@
     "@react-native/gradle-plugin": "^0.72.5",
     "@react-native/js-polyfills": "^0.72.1",
     "@react-native/normalize-colors": "^0.72.0",
-    "@react-native/virtualized-lists": "^0.72.2",
+    "@react-native-mac/virtualized-lists": "^0.0.1",
```

Comment:
Before publishing 0.72, it would be nice to align this to 0.72.x like the upstream one. Not blocking for this review.

Reply:
I'll mark this down as a TODO.